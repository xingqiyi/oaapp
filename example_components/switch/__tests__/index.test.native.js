import React from 'react';
import renderer from 'react-test-renderer';
import Switch from '../index';

describe('Switch', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Switch checked />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
