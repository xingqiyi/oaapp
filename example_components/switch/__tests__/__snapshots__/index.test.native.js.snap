// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Switch renders correctly 1`] = `
<RCTSwitch
  disabled={false}
  onChange={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  onTintColor="#4dd865"
  onValueChange={[Function]}
  style={
    Array [
      Object {
        "height": 31,
        "width": 51,
      },
      undefined,
    ]
  }
  value={true}
/>
`;
